infrastructure/src/main.py

```diff
@@ -0,0 +1,174 @@
+                    "arn:aws:iam::060170161162:role/tamsin.green@bbc.co.uk",
+                    "arn:aws:iam::060170161162:role/lei.he01@bbc.co.uk",
+                ],
+            ),
+            Action=[
+                Action("kms", "Create*"),
+                Action("kms", "Describe*"),
+                Action("kms", "Enable*"),
+                Action("kms", "List*"),
+                Action("kms", "Put*"),
+                Action("kms", "Update*"),
+                Action("kms", "Revoke*"),
+                Action("kms", "Disable*"),
+                Action("kms", "Get*"),
+                Action("kms", "Delete*"),
+                Action("kms", "TagResource*"),
+                Action("kms", "UntagResource*"),
+                Action("kms", "ScheduleKeyDeletion*"),
+                Action("kms", "CancelKeyDeletion*"),
+            ],
+            Resource=["*"],
+        ),
+        Statement(
+            Sid="Allow use of the key",
+            Effect=Allow,
+            Principal=Principal(
+                "AWS",
+                [
+                    "arn:aws:iam::060170161162:role/anna.blaziak@bbc.co.uk",
+                    "arn:aws:iam::060170161162:role/allison.shultes@bbc.co.uk",
+                    "arn:aws:iam::060170161162:role/sarah.rainbow@bbc.co.uk",
+                ],
+            ),
+            Action=[
+                Action("kms", "Encrypt*"),
+                Action("kms", "Decrypt*"),
+                Action("kms", "ReEncrypt*"),
+                Action("kms", "GenerateDataKey*"),
+                Action("kms", "DescribeKey"),
+            ],
+            Resource=["*"],
+        ),
+        Statement(
+            Sid="Allow attachment of persistent resources",
+            Effect=Allow,
+            Principal=Principal(
+                "AWS",
+                [
+                    "arn:aws:iam::060170161162:role/anna.blaziak@bbc.co.uk",
+                    "arn:aws:iam::060170161162:role/allison.shultes@bbc.co.uk",
+                    "arn:aws:iam::060170161162:role/sarah.rainbow@bbc.co.uk",
+                ],
+            ),
+            Action=[
+                Action("kms", "CreateGrant"),
+                Action("kms", "ListGrants"),
+                Action("kms", "RevokeGrant"),
+            ],
+            Resource=["*"],
+            Condition=Condition(Bool({"kms:GrantIsForAWSResource": True})),
+        ),
+    ],
+)
+
+kms_key = t.add_resource(
+    Key(
+        "KmsKey",
+        Description="Key for encrypting the secrets",
+        KeyPolicy=key_policy,
+    )
+)
+
+t.add_output(
+    Output(
+        "KeyId",
+        Value=Ref(kms_key),
+        Description="KeyId of KMS",
+    )
+)
+
+
+kms_alias = t.add_resource(
+    Alias(
+        "KeyAlias",
+        AliasName=Join(
+            "",
+            [
+                "alias/",
+                Join(
+                    "-",
+                    [Ref(t.parameters["Environment"]), Ref(component_name)],
```

Comment:
```suggestion
                    [Ref(component_name)],
```
Should we just have the one kms resource since we will be pulling in both test and live react-app configs at build time? That way we can reference it in the makefile without using the `live` prefix